projects/plugins/jetpack/modules/notes.php

```diff
@@ -136,7 +136,7 @@ public function styles_and_scripts() {
 		$script_handles = array();
 		wp_register_script( 'wpcom-notes-common', $this->wpcom_static_url( '/wp-content/mu-plugins/notes/notes-common-lite.min.js' ), array(), JETPACK_NOTES__CACHE_BUSTER, true );
 		$script_handles[] = 'wpcom-notes-common';
-		wp_enqueue_script( 'wpcom-notes-admin-bar', $this->wpcom_static_url( '/wp-content/mu-plugins/notes/admin-bar-v2.js' ), array( 'wpcom-notes-common' ), JETPACK_NOTES__CACHE_BUSTER, true );
+		wp_enqueue_script( 'wpcom-notes-admin-bar', $this->wpcom_static_url( '/wp-content/mu-plugins/notes/admin-bar-v2.js' ), array( 'wpcom-notes-common' ), JETPACK_NOTES__CACHE_BUSTER . '-' . time() , true );
```

Comment:
Is this also for testing and removed before merging?

Reply:
Yep. I'll update patch now.. its not needed in testing anyway, you can define `JETPACK_NOTES__CACHE_BUSTER` in 0-sandbox.php instead.